docs/site/DEVELOPING.md

```diff
@@ -80,6 +81,25 @@ Please note that we are automatically running the build from `pretest` script,
 therefore you should not need to run this command as part of your
 [red-green-refactor cycle](http://www.jamesshore.com/Blog/Red-Green-Refactor.html).
 
+## Utilizing the monorepo
+
+The monorepo's source code can be utilized for testing new features or bug fixes
+withing LoopBack itself or applications are dependent on LoopBack.
+
+### Using monorepo packages as dependencies
+
+The `/sandbox` directory in the monorepo can be used to utilize the source code
+as symbolically-linked dependencies. See the
+[README](https://github.com/strongloop/loopback-next/blob/master/sandbox/README.md)
+for usage instructions.
+
+### Using the CLI via the monorepo
+
+The CLI can be executed from `/loopback-next/packages/cli/bin/cli-main.js`.
```

Comment:
Can you add a direct link to this file? And show how it can be executed?

Reply:
Suggestion:

The CLI can be invoked from the local `loopback-next` git repo: 

```sh
./loopback-next/packages/cli/bin/cli-main.js
```

The arguments are the same as `lb4`, for example, `./loopback-next/packages/cli/bin/cli-main.js model` is the same as `lb4 model`.